awx/settings/development.py

```diff
@@ -112,14 +112,17 @@
     PEP8_RCFILE = "setup.cfg"
     PYLINT_RCFILE = ".pylintrc"
 
+
+# debug toolbar and swagger assume that requirements/requirements_dev.txt are installed
+
 INSTALLED_APPS += [   # NOQA
     'rest_framework_swagger',
     'debug_toolbar',
 ]
 
-MIDDLEWARE += [  # NOQA
+MIDDLEWARE = [  # NOQA
     'debug_toolbar.middleware.DebugToolbarMiddleware',
-]
+] + MIDDLEWARE
```

Comment:
@beeankha I switched this from the bottom of the stack to the top of the stack. The reason is that the requests made by the other middleware classes will not get profiled unless it's at the top of the stack, and we don't want to ignore performance problems from those.